emails/tests/models_tests.py

```diff
@@ -185,27 +186,30 @@ def test_create_assigns_to_user(self):
 
     @override_settings(MAX_NUM_FREE_ALIASES=5, MAX_ADDRESS_CREATION_PER_DAY=10)
     def test_create_has_limit(self) -> None:
-        for _ in range(10):
-            RelayAddress.objects.create(user=self.premium_user_profile.user)
+        baker.make(RelayAddress, user=self.premium_user, _quantity=10)
```

Comment:
suggestion (blocking):
```suggestion
        baker.make(RelayAddress, user=self.premium_user, _quantity=MAX_ADDRESS_CREATION_PER_DAY)
```